src/config/network/iwd.md

```diff
@@ -0,0 +1,119 @@
+  Commands                                          Description
+--------------------------------------------------------------------------------
+
+Adapters:
+  adapter list                                      List adapters
+  adapter <phy> show                                Show adapter info
+  adapter <phy> set-property <name> <value>         Set property
+...
+```
+
+Some useful commands:
+
+```
+[iwd]# device list                                       # List all Wi-Fi devices
+[iwd]# device <interface> show                           # Display details of a Wi-Fi device 
+[iwd]# station <interface> scan                          # Scan for networks
+[iwd]# station <interface> get-networks                  # List networks
+[iwd]# station <interface> connect <network-name>        # Connect to a network
+[iwd]# station <interface> connect-hidden <network-name> # Connect to a hidden network
+[iwd]# station <interface> disconnect                    # Disconnect from a network
+[iwd]# known-networks list                               # List known networks
+[iwd]# known-networks forget <network-name>              # Forget a known network
+```
+
+> Note: `iwctl` also supports passing commands as arguments.
+> 
+> ```
+> $ iwctl station <interface> get-networks
+> ```
+
+## Daemon configuration
+
+The main configuration file is located in `/etc/iwd/main.conf`. If it does not exist, you may create it.
+
+### Disable periodic scan for available networks
+
+By default when IWD is in the disconnected state, it periodically scans for available networks. This behaviour may be configured by setting the value of `disable_periodic_scan` to `true` or `false` (default).
+
+```
+[Scan]
+disable_periodic_scan=true
+```
+
+## Network configuration
+
+IWD stores information on known networks, and reads information on pre-provisioned networks from text configuration files located in `/var/lib/iwd`. You can create, modify or remove those files. IWD monitors the directory for changes.
+
+> Note: Files inside the `/var/lib/iwd` directory have permssions set to be read and written only by the root user.
+
+File names are based on the network's SSID and security type: Open, PSK-protected or 802.1x. The name consist of the encoding of the SSID followed by .open, .psk or .8021x.
+
+A simple example of a WPA2/PSK secured network configuration at `/var/lib/iwd/<ssid>.psk` contains only the pre-shared key and the plaintext password.
+
+```
+[Security]
+PreSharedKey=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+Passphrase=<password>
+```
+
+> Note: If the plaintext password is included and the pre-shared key is ommited, the pre-shared key will automatically be calculated from the SSID and password, and written to this file by IWD upon attempting to connect. If the pre-shared key is included and the plaintext password ommited, the network will connect and the plaintext password will not be written to the network configuration file.
```

Comment:
I was not too sure about the clarity of my wording on this note. Open to suggestions on improving this.